addons/website/static/src/snippets/s_social_media/000.scss

```diff
@@ -13,7 +15,17 @@
             justify-content: center;
             align-items: center;
         }
+        i.fa:not(.fa-2x, .fa-3x, .fa-4x, .fa-5x) {
+            &.rounded-circle,
+            &.rounded {
+                width: $-size - .5rem;
+                height: $-size - .5rem;
+            }
+        }
         margin: .2rem;
+        &:has(i.fa-stack:not(.fa-2x, .fa-3x, .fa-4x, .fa-5x)) {
+            margin: .2rem 0;
```

Comment:
Do we really need the vertical margin here?
else i guess we can just do, ( I believe for other size too we should remove margin )
```suggestion
            &:not(i.fa-stack) {
                margin: .2rem;
            }
```

For medium size,

1. Without margin, 
![2](https://github.com/user-attachments/assets/25ab501c-82a0-4b80-a9dc-007b977d2e87)

2. With margin .2rem
![1](https://github.com/user-attachments/assets/6b5bb64d-05c6-45d6-9887-7d146404580b)